packages/react/tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "hedwig-tsconfig/react-library.json",
   "include": ["."],
-  "exclude": ["dist", "build", "node_modules"]
+  "exclude": ["dist", "build", "node_modules"],
+  "compilerOptions": {
+    "verbatimModuleSyntax": true
+  }
```

Comment:
❓ Hvilket problem løser denne?

Reply:
Sånn jeg har forstått det gjør denne reglene enklere for compileren når det gjelder imports og exports og hva som skal beholdes i koden og ikke. Med denne `true` er det `type` keyword som er fasit, i stedet for at den må prøve å gjette om ting er en type eller ikke.
Det er i tråd med linten vår som sier at types skal eksporteres og importeres eksplisitt med dette keywordet.

Rent praktisk gjør det at IntelliJ automatisk skjønner at den skal importere med `type` keyword, så man slipper å få lint-feil, og så måtte legge det til selv.
Ja, jeg vet at lint fix on save er en greie, men jeg er ikke så glad i at det skjer greier on save, og dette føles som en mer riktig måte å oppnå dette på 😄

Reply:
Aha. Takk